### PR TITLE
[hi] Localize /en/docs/reference/glossary/kubectl.md

### PR DESCRIPTION
**Pull request by sachinparihar**
Fixes: #37516

### CODE REVIEW

**Comment by bishal7679** [comment]
Fixes: #35600

File: content/hi/docs/reference/glossary/kubectl.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+- tool
+- fundamental
+---
+कुबेरनेट्स API का उपयोग करके कुबेरनेट्स क्लस्टर 
+{{< glossary_tooltip text="कंट्रोल प्लेन" term_id="control-plane">}} 
+के साथ बातचीत करने के लिए कमांड लाइन टूल।
+
+<!--more-->
+
+आप `kubectl` का उपयोग कुबेरनेट्स ऑब्जेक्ट बनाने, निरीक्षण करने, अपडेट करने और हटाने के लिए कर सकते हैं।
```
Comment (bishal7679) [inline content/hi/docs/reference/glossary/kubectl.md:21]:
```suggestion
आप कुबेरनेट्स ऑब्जेक्ट बनाने, निरीक्षण करने, अपडेट करने और हटाने के लिए `kubectl` का उपयोग कर सकते हैं।
```

**Review by bishal7679 — CHANGES_REQUESTED** [review]
Suggested some changes! commit this suggestion if you found this useful

File: content/hi/docs/reference/glossary/kubectl.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+title: kubectl
+id: kubectl
+date: 2018-04-12
+full_link: /docs/user-guide/kubectl-overview/
+short_description: >
+  कुबेरनेट्स क्लस्टर के साथ बातचीत करने के लिए एक कमांड लाइन टूल।
```
[the remark this answers is not in the document]
Comment (divya-mohan0209) [reply content/hi/docs/reference/glossary/kubectl.md]: Tbh, संचार fits better in this case.

File: content/hi/docs/reference/glossary/kubectl.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+aka:
+- kubectl
+tags:
+- tool
+- fundamental
+---
+कुबेरनेट्स API का उपयोग करके कुबेरनेट्स क्लस्टर 
+{{< glossary_tooltip text="कंट्रोल प्लेन" term_id="control-plane">}} 
+के साथ बातचीत करने के लिए कमांड लाइन टूल।
```
[the remark this answers is not in the document]
Comment (divya-mohan0209) [reply content/hi/docs/reference/glossary/kubectl.md]: The original text worked better here too. We refer to Control Plane in the reference docs in the former format. Please refer: https://github.com/kubernetes/website/blob/main/content/hi/docs/reference/glossary/control-plane.md

**Comment by divya-mohan0209** [comment]
Hello @sachinparihar , please address the feedback provided & resolve the merge conflict so that we can ready this for merge.

**Comment by sachinparihar** [comment]
@divya-mohan0209 please help me!

**Comment by divya-mohan0209** [comment]
@sachinparihar:

1. Sign the LF CLA before you submit a PR to the kubernetes/website repo. Make sure you have added the email ID & username that you've used for signing to your local/global variables.
2. If it is simpler, please consider closing this PR & reopening a new PR addressing the changes.

**Comment by sachinparihar** [comment]
new pr #40438

**Pull request closed without merging**